### DISCUSSION TITLE
Software Engineering Survey on GitHub Discussions.

### DISCUSSION THREAD

**Discussion by raux** [gomods/athens#1665] [General] [answered]
Do you find GitHub discussions useful as a communication channel? Maybe not? 
Hello Developers! I am Raula Gaikovina Kula, a Software Engineering researcher
https://raux.github.io/

Together with a team of researchers, we are interested in your thoughts about
using the GitHub Discussions Beta. We noticed that your project is currently using.

We have developed a simple 3 question survey that could take up to 5 mins to complete.
https://forms.gle/peBxiio8u99QvKMk6

Thank you in advance for your time and apologize if this is not a proper use of this channel.

@raux
Raula, PhD on behalf of Hideaki, Sebastian, Nicole and Christoph

**Comment by arschles** [comment, marked as answer]
Hi @raux the project and I haven't used them much recently so I can't really give you an informed answer to that. I noticed that a few questions about the project were recently asked here so maybe that will change in the future.
